src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.impl.ParseTransactionServiceImpl;
+import core.basesyntax.service.impl.ReaderServiceImpl;
+import core.basesyntax.service.impl.ReportCreatorImpl;
+import core.basesyntax.service.impl.WriterServiceImpl;
+import core.basesyntax.strategy.TransactionProcessor;
+import core.basesyntax.strategy.operations.BalanceOperation;
+import core.basesyntax.strategy.operations.OperationHandler;
+import core.basesyntax.strategy.operations.PurchaseOperation;
+import core.basesyntax.strategy.operations.ReturnOperation;
+import core.basesyntax.strategy.operations.SupplyOperation;
+import java.util.HashMap;
+import java.util.List;
+
+public class Main {
+    private static final String PATH_TO_CSV_FILE = "src/main/resources/inputFile.txt";
+    private static final String PATH_TO_CSV_REPORT_FILE = "src/main/resources/recordFile.txt";
+    private static final HashMap<Operation, OperationHandler>
+            operationHandlersStrategy = new HashMap<>();
+
+    static {
+        operationHandlersStrategy.put(Operation.BALANCE, new BalanceOperation());
+        operationHandlersStrategy.put(Operation.PURCHASE, new PurchaseOperation());
+        operationHandlersStrategy.put(Operation.RETURN, new ReturnOperation());
+        operationHandlersStrategy.put(Operation.SUPPLY, new SupplyOperation());
+    }
+
+    public static void main(String[] args) {
+        ReaderServiceImpl reader = new ReaderServiceImpl();
```

Comment:
```suggestion
        ReaderService reader = new ReaderServiceImpl();
```
use abstraction. 
**Fix it everywhere!**